Dockerfile

```diff
@@ -0,0 +1,26 @@
+﻿FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine
+
+EXPOSE 5000
+EXPOSE 5001
+
+ARG USER=dev
+ARG DUID=1000
+ARG DGID=1000
+ARG ISMAC=0
+
+VOLUME [ "/src" ]
+WORKDIR /src
+
+RUN if [ ${ISMAC} -eq 0 ]; then \
+    addgroup -g ${DGID} -S ${USER}; \
+    fi
+RUN adduser -S -G ${USER} -u ${DUID} -s /bin/bash ${USER}
```

Comment:
Even the `addgroup` is not called now, the `-G GROUP` part still produces an error.

Reply:
Changing it to a static value can fix the build, the problem is because the mac user group id is pre existing in the alpine image - so a new group cannot be created with it.
Changing it removes the purpose of the user creation (in this way), because setting the id to the same as the host user is to avoid the permission errors due to the dotnet processes file usage.

Changed the user creating to use the preexisting group